peps/pep-0007.rst

```diff
@@ -26,12 +26,21 @@ particular rule:
    clean up someone else's mess (in true XP style).
 
 
-C dialect
-=========
+C standards
+===========
+
+Follow the following standards.
+For features that aren't in the relevant standard (like atomics or some C11
+features in public headers), use CPython-specific wrappers.
+When adding such wrappers, try to make them easy to adjust for unsupported
+compilers.
 
 * Python 3.11 and newer versions use C11 without `optional features
   <https://en.wikipedia.org/wiki/C11_%28C_standard_revision%29#Optional_features>`_.
```

Comment:
The advice is to use anonymous links unless the document reuses the target name:
```suggestion
  <https://en.wikipedia.org/wiki/C11_%28C_standard_revision%29#Optional_features>`__.
```